python/users.py

```diff
@@ -17,6 +17,19 @@
 from sdk.python.domains import EnrolledDomain
 
 
+class SystemRole(sdk.python.entities.Entity):
+
+    resource = '/system_roles/'
+    json_name = 'system_role'
+
+    def __init__(self, **kwargs):
+        super(SystemRole, self).__init__(**kwargs)
```

Comment:
i think we can just have `super()`... in python3

Reply:
agreed